server/local/studentdashboard-LOCALHOST.yml

```diff
@@ -55,7 +59,7 @@ latte_admin_group: TL-Latte-admin-test
 ## from local files.  Configuration is documented below.
 
 ### file based data provider value:
-data_provider_file_directory: server/test-files/courses
+data_provider_file_directory: server/test-files
```

Comment:
No.  The test files now come from two different sub-directories so this is specifying the parent directory.